npm/pkg/dataplane/policies/policymanager.go

```diff
@@ -40,6 +52,10 @@ func (pMgr *PolicyManager) Initialize() error {
 	return nil
 }
 
+type PolicyManagerCfg struct {
```

Comment:
note to add this in Constructor for pMgr in another PR (will touch code in dataplane.go, dataplane_test.go though)